code/cmdline/cmdline.cpp

```diff
@@ -516,6 +516,7 @@ cmdline_parm log_to_stdout_arg("-stdout_log", nullptr, AT_NONE); // Cmdline_log_
 cmdline_parm slow_frames_ok_arg("-slow_frames_ok", nullptr, AT_NONE);	// Cmdline_slow_frames_ok
 cmdline_parm fixed_seed_rand("-seed", nullptr, AT_INT);	// Cmdline_rng_seed,Cmdline_reuse_rng_seed;
 cmdline_parm luadev_arg("-luadev", "nullptr", AT_NONE);	// Cmdline_lua_devmode
+cmdline_parm override_arg("-override", nullptr, AT_NONE);	// Cmdline_override_data
```

Comment:
This may be better as `-override_data` instead, or anything more descriptive really, just so that it's perfectly clear what is going to happen when this option is used. Otherwise it's a bit vague and could be for anything.